opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -349,6 +349,10 @@ message ValueType {
 // both fields are populated, they MUST contain the same number of elements, and
 // the elements at the same index MUST refer to the same event.
 //
+// For the purposes of efficiently representing aggregated data observations, a Sample is regarded
+// as having a shared identity and an associated collection of per-observation data points.
```

Comment:
I think this is *against* the goal of sharing Resource  and its notion of identity with OpenTelemetry protos.

Reply:
+1 to @christos68k, this change merely improves the documentation around the general rules of what constitutes the identity of a sample in the profiling signal OTLP. It is orthogonal to specific labels, like resource reference, should we introduce that.